core/common/platform_gl.cpp

```diff
@@ -0,0 +1,287 @@
+#include "platform_gl.h"
+
+#include "gl.h"
+#include "gl/error.h"
+
+namespace Tangram {
+
+GLenum GL::getError() {
+    auto result = glGetError();
+    GL_CHECK();
```

Comment:
Good eye!!

On Sep 8, 2016 5:38 AM, "Hannes Janetzek" <notifications@github.com> wrote:

> In core/common/platform_gl.cpp
> <https://github.com/tangrams/tangram-es/pull/955#discussion_r77974894>:
>
> > @@ -0,0 +1,287 @@
> > +#include "platform_gl.h"
> > +
> > +#include "gl.h"
> > +#include "gl/error.h"
> > +
> > +namespace Tangram {
> > +
> > +GLenum GL::getError() {
> > +    auto result = glGetError();
> > +    GL_CHECK();
>
> Good catch!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/955/files/ca29183f72c40b8410f5a6d9eee814d85a59248c#r77974894>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAWAwfIFQmQNBWKnJIobH6sz-fIHQNbCks5qn9edgaJpZM4J2sqj>
> .
>